client/test/diagnostics/annotation_3.ion

```diff
@@ -0,0 +1 @@
+false::23
```

Comment:
Why is this invalid?

Reply:
I am wondering as well. However, It is in the bad data samples. The link is here: [https://github.com/amzn/ion-tests/blob/master/iontestdata/bad/annotationFalse.ion]. (https://github.com/amzn/ion-tests/blob/master/iontestdata/bad/annotationFalse.ion)

Reply:
I am wondering as well. However, It is in the bad data samples. The link is here: https://github.com/amzn/ion-tests/blob/master/iontestdata/bad/annotationFalse.ion